docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
```

Comment:
I also made a small mistake... the we don't need to get an IP for the exact node that the apiserver is running on. If the service type is the default specified in the chart (`NodePort`), then port `30080` should end up being reserved on _all_ nodes and the kube-proxy should ensure all traffic received on that port gets to where it's got to go. So you just need the IP of _any_ node in your cluster. (It's worth double-checking me on this.)

Wording could be something like:

To configure `kubectl` to communicate with the service catalog API server, we'll have to find an IP address where the apiserver is reachable from outside your cluster. If you installed the chart without tweaking the apiserver's service type, then it is one of type `NodePort`. You may communicate with the apiserver using the IP of _any_ node in the cluster and the port `30080`.